protocol/message_persistence.go

```diff
@@ -3075,15 +3075,7 @@ func (db sqlitePersistence) findAndUpdateReplies(tx *sql.Tx, bridgeMessageID str
 }
 
 func (db sqlitePersistence) findAndUpdateRepliedTo(tx *sql.Tx, discordParentMessageID string, statusMessageID string) error {
-	// Finds status messages id which are replies for bridgeMessageID
-	repliedMessageID, err := db.findStatusMessageIDForBridgeMessageID(tx, discordParentMessageID)
```

Comment:
nice! we could also nuke this function definition then `findStatusMessageIDForBridgeMessageID `